generator.yaml

```diff
@@ -19,8 +19,15 @@ resources:
         is_required: false
     update_operation:
       custom_method_name: customUpdateApi
+  DomainName:
+    fields:
+      DomainName:
```

Comment:
Maybe this isn't the right place for this, but ideally I'd prefer this field to be called `Name`. This is inline with `BucketName` changing to `Name` in S3, and a host of other places where we remove the redundant prefix of field names.

Reply:
@RedbackThomson kind of agree... but in this particular case, it's the resource itself that is called `DomainName` as opposed to `Domain` :/ One wonders why this is a resource as opposed to a field/attribute on another resource... but :man_shrugging:

Reply:
Yeah, I will handle it in separate issue.